src/assets/js/app.js

```diff
@@ -220,14 +220,14 @@ $(document).ready(function(){
         })
     }
 
-    // smooth scrolling to anchor tag when clicking anchor link
```

Comment:
This was deactivated anyways (due to #440), right?

Reply:
Yep it was quite a while back